### PR TITLE
[DRAFT] Fix readme repo level tf

### PR DESCRIPTION
**Pull request by shivdeep-singh-ibm**

### CODE REVIEW

File: transforms/code/repo_level_ordering/ray/README.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ testing and IDE set up.
 
 ## Summary 
 
+This transform ccan group the data by `repo_name` and apply additional transformations
```
Comment (Param-S) [inline transforms/code/repo_level_ordering/ray/README.md:10]: `ccan` - typo

File: transforms/code/repo_level_ordering/ray/README.md [hunk 2]
```diff
@@ -79,81 +82,76 @@ To see results of the transform.
...
+python repo_level_order_transform_ray.py  \
+             --data_s3_cred "$s3_kreds" \
+             --data_s3_config "$s3_conf" \
+             --runtime_num_workers 80 \
+             --repo_lvl_store_type "ray" \  
+             --repo_lvl_store_ray_cpus 0.2 \
+             --repo_lvl_store_ray_nworkers 10 \
+             --repo_lvl_sorting_enabled  \
+             --repo_lvl_output_by_langs 
+
```
Comment (daw3rd) [inline transforms/code/repo_level_ordering/ray/README.md:156]: Can you add the short section on running as an image. For example, https://github.com/IBM/data-prep-kit/blob/dev/transforms/universal/noop/ray/README.md#transforming-data-using-the-transform-image

**Review by daw3rd — CHANGES_REQUESTED** [review]

File: transforms/code/repo_level_ordering/ray/README.md [hunk 3]
```diff
@@ -79,81 +82,76 @@ To see results of the transform.
...
-)
+2. Running on Cluster
 
-```sh
-export INPUT_FOLDER="input_data_folder/"
-export OUTPUT_FOLDER="output_data_folder/"
+This configuration can be used when data is on S3 like COS storage and computation is done
+on a ray cluster.
 
+Recommended `repo_lvl_store_type` for cluster is 'ray'. We need to dedicate some ray actors specially for
```
Comment (Param-S) [inline transforms/code/repo_level_ordering/ray/README.md:118]: can we describe more about this option `repo_lvl_store_type` in section `Configuration and command line Options`. 

If we include details which describe the list of supported values and its usage/application.

File: transforms/code/repo_level_ordering/ray/README.md [hunk 4]
```diff
@@ -79,81 +82,76 @@ To see results of the transform.
...
+ray store.
 
-local_conf="{'input_folder' : '$INPUT_FOLDER', 'output_folder' : '$OUTPUT_FOLDER'  }"
+for example, if we have a ray cluster with 100 actors available. We can dedicate some for the store. Let us randomly
+use 80 for computation and 10 for store and let 10 remain free.
```
Comment (Param-S) [inline transforms/code/repo_level_ordering/ray/README.md:122]: What is the reason for leaving 10 actors free? Whats the logic behind this example actors allocation? (based on the input data volume or data/repo distribution)?

**Review by Param-S — CHANGES_REQUESTED** [review]

**Comment by daw3rd** [comment]
Can you also add this transform to the table in the top level README.md

**Comment by shivdeep-singh-ibm** [comment]
readme changes as part in https://github.com/IBM/data-prep-kit/pull/434

**Pull request closed without merging**